File: fix_mobile.js
const fs = require('fs');
let code = fs.readFileSync('app/page.jsx', 'utf8');

// Replace HeroSection ID
code = code.replace('<section ref={secRef} style={{', '<section id="hero-section" ref={secRef} style={{');

fs.writeFileSync('app/page.jsx', code);
console.log("Fixed page.jsx");

File: fix_mobile.py
with open("app/page.jsx", "r") as f:
    page_code = f.read()

page_code = page_code.replace('<section ref={secRef} style={{', '<section id="hero-section" ref={secRef} style={{')

with open("app/page.jsx", "w") as f:
    f.write(page_code)
